mycroft/res/text/de-de/not connected to the internet.dialog

```diff
@@ -0,0 +1,5 @@
+Es sieht so aus, als wäre ich nicht mit dem Internet verbunden
+Es scheint als wäre ich nicht mit dem Internet verbunden
+Ich kann das Internet gerade nicht erreichen
+Es ist mir gerade nicht möglich das das Internet zu erreichen
+
```

Comment:
Empty lines like this will also be considered an alternative when selecting the output line causing Mycroft to be silent when speaking the dialog and should therefore be removed.